build-aux/com.github.melix99.telegrand.Devel.json

```diff
@@ -73,17 +75,13 @@
         },
         {
             "name": "tdlib",
-            "buildsystem": "simple",
-            "build-commands": [
-                "install -D libtdjson.so.* /app/lib",
-                "ln -sf /app/lib/libtdjson.so.* /app/lib/libtdjson.so",
-                "install -D pkgconfig/* /app/lib/pkgconfig"
-            ],
+            "buildsystem": "cmake-ninja",
```

Comment:
Just for my understanding: 
I tried building the flatpak and it was amazingly fast compared to building it with cmake from the command line
with `-DCMAKE_BUILD_TYPE:STRING=Release`. When this option is not specified, it defaults to a debug build, right?

I'm fine with using a debug build but I just want to be sure whether you are aware of that.

Reply:
Hm, you're right, actually I forgot to set the the build type to release here.